src/components/ProjectsPage.js

```diff
@@ -1,83 +1,121 @@
 import React from "react";
+import { Button, Row, Col, Card, Container } from 'react-bootstrap';
+import "../styling/projects-page-styling.css";
 
 let companies = [{id: 0, title: "Project Title 1", name: "Organization Name 1"},
                  {id: 1, title: "Project Title 2", name: "Organization Name 2"},
                  {id: 2, title: "Project Title 3", name: "Organization Name 3"},
                  {id: 3, title: "Project Title 4", name: "Organization Name 4"},
-                 {id: 4, title: "Project Title 5", name: "Organization Name 5"}]
+                 {id: 4, title: "Project Title 5", name: "Organization Name 5"},
+                 {id: 5, title: "Project Title 6", name: "Organization Name 6"},
+                 {id: 6, title: "Project Title 7", name: "Organization Name 7"}]
+
 
 class ProjectsPage extends React.Component {
 
     state = {
         selected: -1
     }
 
-
     render() {
         return (
-          <div class="page-container">
 
-            <div class="row">
-                <div class="col col-md-7 overflow-auto scrolling">
-                    <div class="row card-row">
+            <React.Fragment>
```

Comment:
tip: if you write a line like `const { Body, Title, Subtitle } = Card` above the render fxn then you won't need to keep writing `Card.` in all the tags. i hate to type lol same w fragment, in the import you can just do `import { Fragment }, React from 'react'`

Reply:
omg this is great to know! updated in code.